src/mailmojo-widget.php

```diff
@@ -1,5 +1,5 @@
 <?php
-/*  Copyright 2013  Eliksir AS  (email : post@e5r.no)
+/*  Copyright 2017  Eliksir AS  (email : post@e5r.no)
```

Comment:
Are we in 2017 already? ;) Anyhow, having year here is pretty pointless, just get rid of the year (and in other files) for a general copyright.